conan/tools/cmake/cmake.py

```diff
@@ -112,9 +113,9 @@ def configure(self, variables=None, build_script_folder=None, cli_args=None):
 
         command = " ".join(arg_list)
         with chdir(self, build_folder):
-            self._conanfile.run(command)
+            self._run(command, redirect_stdout=redirect_stdout)
```

Comment:
Maybe we want the ``conanfile.run(..., stdout_file=<filepath>)`` interface that avoids the context ``with open() as f`` to pass it to ``conanfile.run()``?